src/Applications/LDAS_App/GEOSldas_LDAS.rc

```diff
@@ -181,3 +181,7 @@ DYCORE: none
 # ---- For MAPL_RestartOptional
 #
 MAPL_ENABLE_BOOTSTRAP: YES
+
+#----- Write restart or checkpoint by oserver
+#
```

Comment:
I don't quite understand the connection between setting WRITE_RESTART_BY_OSERVER to NO here and the specifications in the batch inputs file.
I suppose the NO here is just the default and will be overwritten by YES during setup if the user requests oserver_nodes>0 in the batch inputs file.
But what happens if the user simply edits LDAS.rc after setup and changes NO to YES?  Would that work?  I feel we need to put some instructions on how WRITE_RESTART_BY_OSERVER works.
Also, the name WRITE_RESTART_BY_OSERVER suggests that only restarts (and checkpoint files) are written using the oserver, but I thought the oserver also handles MAPL History.  Is there a difference between the two output streams?  
No need to write a long reply.  We can discuss at a forthcoming meeting.

Reply:
I should change it to NO by default.